frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/PaymentsInputPanel.tsx

```diff
@@ -156,117 +152,103 @@ const PaymentInput = ({ isDisabled }: { isDisabled: boolean }) => {
 
     // update sessionStorage to enable admin feedback
     setisAdminFeedbackEligible(true)
-    return paymentsMutation.mutate(paymentsData, {
-      onSuccess: () => {
-        setToInactive()
-        handleClose()
+    return paymentsMutation.mutate(
+      { ...paymentsData, enabled: true },
+      {
+        onSuccess: () => {
+          setToInactive()
+          handleClose()
+        },
       },
-    })
+    )
   })
 
   return (
     <CreatePageDrawerContentContainer>
       <FormControl
+        isRequired
+        isDisabled={isDisabled}
         isReadOnly={paymentsMutation.isLoading}
+      >
+        <FormLabel>Payment type</FormLabel>
+        <Controller
+          name={'payment_type'}
+          control={control}
+          render={({ field }) => (
+            <SingleSelect
+              isClearable={false}
+              placeholder="Select Payment Type"
+              fullWidth={true}
```

Comment:
/nit not related to your changes, but could you help me to simply `fullWidth={true}` to `fullWidth`?

Reply:
Thanks Ken! [cfb6493](https://github.com/opengovsg/FormSG/pull/6511/commits/cfb64937fe2bbc615ff20a389dfe687a21e43b60)